tests/openvino/test_modeling.py

```diff
@@ -1320,17 +1324,17 @@ def test_compare_with_and_without_past_key_values(self):
 
 class OVModelForAudioClassificationIntegrationTest(unittest.TestCase):
     SUPPORTED_ARCHITECTURES = (
-        # "audio_spectrogram_transformer",
-        # "data2vec_audio",
-        # "hubert",
-        # "sew",
-        # "sew_d",
-        # "wav2vec2-conformer",
+        "audio_spectrogram_transformer",
+        "data2vec_audio",
+        "hubert",
+        "sew",
+        "sew_d",
```

Comment:
I see a couple of new errors in the CI related to these models. Is there a reason why they're uncommented in this PR ?

Reply:
added them to know what we can have in the supported architectures [section]( https://moon-ci-docs.huggingface.co/docs/optimum-intel/pr_765/en/openvino/architectures)